tests/optimize_onnx_test.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import tarfile
+import tempfile
+import unittest
+
+from subprocess import Popen, PIPE
+from six.moves.urllib.request import urlretrieve
+import numpy as np
+
+import onnx
+from onnx import helper, ModelProto, TensorProto
+
+import onnx_caffe2.backend as c2
+
+from tests.test_utils import TestCase
+
+# mostly copied from
+# https://github.com/onnx/onnx/blob/master/onnx/backend/test/runner/__init__.py
+
+class TestRoundtrip(TestCase):
+    def _roundtrip(self, model_name):
```

Comment:
Why not invoke _prepare_model_data
https://github.com/onnx/onnx/blob/master/onnx/backend/test/runner/__init__.py